docs/source/key_components/dashboard_visualization.rst

```diff
@@ -0,0 +1,186 @@
+Dashboard Visualization
+=======================
+
+Env-dashboard is a post-experimental visualization tool, aims to provide
+more intuitive environment information, which will guide the design of
+the algorithm and continually fine-tuning.
+
+Currently, the visualization of senario **Container Inventory Management**
+and **Citi Bike** are supported.
+
+Dependency
+----------
+
+Module **streamlit** and **altair** should be pre-installed.
+
+* `streamlit <https://www.streamlit.io/>`_
+
+
+* `altair <https://altair-viz.github.io/>`_
+
+Install them with:
+
+.. code-block:: sh
+
+    pip install streamlit altair
+
+----
+
+How to Use?
+-----------
+
+Get Data
+~~~~~~~~
+
+Experimental data is dumped automatically while running the experiment.
+User need to run experiment through the file in folder maro/examples in source code. 
+User could specify dump destination folder by setting the parameter **opts** as below:
+
+.. code-block:: sh
+
+    opts['enable-dump-snapshot'] = EXPECTED_OUTPUT_FOLDER
```

Comment:
we also need code snippets for enabling dump and changing dump target folder.

@Meroy9819 Do you know how to run based on these?